test-e2e/cip-auditor/cip-auditor-e2e.go

```diff
@@ -324,25 +307,23 @@ func populateGoldenImages(repoRoot string) error {
 }
 
 // TODO: De-dupe with other e2e functions
-func getBazelOption(repoRoot, o string) string {
+func getWorkspaceStatus(repoRoot string) map[string]string {
+	fmt.Println("Reading workspace variables")
+
+	status := make(map[string]string)
 	cmd := command.NewWithWorkDir(repoRoot, "./workspace_status.sh")
 	std, err := cmd.RunSuccessOutput()
 	if err != nil {
 		klog.Errorln(err)
-		return ""
+		return status
 	}
 
 	stdout := std.Output()
 	for _, line := range strings.Split(strings.TrimSuffix(stdout, "\n"), "\n") {
-		if strings.Contains(line, o) {
-			words := strings.Split(line, " ")
-			// nolint[gomnd]
-			if len(words) == 2 {
-				return words[1]
-			}
-		}
+		words := strings.Split(line, " ")
+		status[words[0]] = words[1]
```

Comment:
Nit: Is there some way to bubble up an error if `words` doesn't have exactly 2 elements? The current approach is not any worse than the previous behavior, but it's worth considering how to avoid the hardcoded element access.

Reply:
Ack. Added the following check:
```
if len(words) != 2 {
	klog.Warningf("Unexpected key value pair in line: %d!\n", idx)
}
```

Reply:
It would be simpler to just log and exit with an error. I think `klog.Fatalf()` should do the trick IIRC

Reply:
Also, when you log the troublesome line, it's better to log the actual line contents. The way you have it now, you're logging an integer (line number) so the user has to do the lookup themselves which is rather troublesome.

Reply:
@tylerferrara Instead of logging `idx`, can you log the contents of the line, using the `line` variable?

Reply:
Merged before I even saw this. Fixing in #319